### DISCUSSION TITLE
Home Page Favorites

### DISCUSSION THREAD

**Discussion by abelokoj-github** [sumatrapdfreader/sumatrapdf#3077] [Ideas]
I will like to have an option that allows me to pin certain documents the home page of Sumatra pdf as favorite documents so I can easily click and open them whenever I want.

Also, they should give the user the ability to select the maximum amount of favorite documents that should appear in the home page. I think this will be very helpful.

Something like this below but you should be allowed to pin selected documents of your own choice. ** It should allow me pin selected documents in the homepage for quicker access  instead of using just the recent files menu.** I think this might be a good replacement for frequently accessed documents.

**There should also be the option from file menu to click and say go to (open) the  homepage tab**

![image](https://user-images.githubusercontent.com/69094732/199842954-5c6a5bd5-0f73-41fc-abc8-805c7a33184a.png)

**Comment by GitHubRulesOK** [comment]
pin document has been there for the maximum permitted <kbd>0</kbd> to <kbd>9</kbd> number key entries as long as I can remember there is no <kbd>10</kbd> on the keyboard only F10

![image](https://user-images.githubusercontent.com/12754913/199871356-b9d5edb7-0ace-49a8-a093-175c593c3e4c.png)

**Reply by abelokoj-github** [reply]
Alright, However, I will appreciate if the user can be able to increase the number of files that can be pinned say like up to 50.

Also, if possible, how can we set a shortcut to pin current document to favorites page?

Thanks in anticipation.

**Reply by alexandrmalinovsky** [reply]
I'm asking for the same. Ability to pin and to set more than 10 documents on the home page.

**Reply by goyalyashpal** [reply]
hey @GitHubRulesOK how did you open the "frequently read" home page in a separate tab along with existing documents?

**Reply by GitHubRulesOK** [reply]
@yashpalgoyal1304 Addition of home page is in Pre-release but showing favourite sidebar is very old settings

**Comment by GitHubRulesOK** [comment]
Well I have suggested in the past you can have as many favourites as practical, that easier with new home tab and modern screens.
The hot key is Ctrl=B 

![image](https://user-images.githubusercontent.com/12754913/200628551-496106ef-9cc7-4302-8a28-7cf670103e9b.png)

***The prime issue is they will only be listed in system file alpha order so for ranking would need external file name changing***

![image](https://user-images.githubusercontent.com/12754913/200630072-ed1eb186-f2c5-4dda-9ce7-6980d33fede1.png)
